tests/ha/drbd_passive.pm

```diff
@@ -21,6 +21,12 @@ use testapi;
 use lockapi;
 use hacluster;
 
+sub is_standalone {
```

Comment:
Subroutine name is a little misleading, as it doesn't check whether it is standalone or not, but assures that it is. It's up to you if want to rename. Other than that LGTM

Reply:
`assert_standalone` maybe?

Reply:
@rwx788 you're right, it's more to be sure that DRBD status is standalone or not, so I used the name suggested by @okurz.